home.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="./assets/styles/homeStyle.css">
+</head>
+<body>
+    <head>
+        <div class="main-container">
+        <div class="mobile-nav">
+            <div class="moblile-menu-icon">
```

Comment:
- [ ] Please note that your website design is missing the top nav containing the social icons. Kindly add them to match the design guideline given to you👍🏽 

![image](https://user-images.githubusercontent.com/63473584/190509000-b6d63def-de39-4cb0-9b10-7e1c19bc8643.png)